packages/grid-pro/test/edit-column.test.js

```diff
@@ -19,307 +18,6 @@ import {
 } from './helpers.js';
 
 describe('edit column', () => {
-  (isIOS ? describe.skip : describe)('keyboard navigation', () => {
```

Comment:
Remove check for `isIOS` as we don't currently have setup for running tests on iOS.